registry_provider_version.go

```diff
@@ -34,15 +34,15 @@ type registryProviderVersions struct {
 
 // RegistryProviderVersion represents a registry provider version
 type RegistryProviderVersion struct {
-	ID                 string                      `jsonapi:"primary,registry-provider-versions"`
-	Version            string                      `jsonapi:"attr,version"`
-	CreatedAt          string                      `jsonapi:"attr,created-at,iso8601"`
-	UpdatedAt          string                      `jsonapi:"attr,updated-at,iso8601"`
-	KeyID              string                      `jsonapi:"attr,key-id"`
-	Protocols          []string                    `jsonapi:"attr,protocols,omitempty"`
-	Permissions        RegistryProviderPermissions `jsonapi:"attr,permissions"`
-	ShasumsUploaded    bool                        `jsonapi:"attr,shasums-uploaded"`
-	ShasumsSigUploaded bool                        `jsonapi:"attr,sasums-sig-uploaded"`
+	ID                 string                             `jsonapi:"primary,registry-provider-versions"`
+	Version            string                             `jsonapi:"attr,version"`
+	CreatedAt          string                             `jsonapi:"attr,created-at,iso8601"`
+	UpdatedAt          string                             `jsonapi:"attr,updated-at,iso8601"`
+	KeyID              string                             `jsonapi:"attr,key-id"`
+	Protocols          []string                           `jsonapi:"attr,protocols,omitempty"`
```

Comment:
omitempty is not necessary here as the `Protocols` field will be initialized to `[]` if the corresponding json field is empty.